ldb/validators.py

```diff
@@ -0,0 +1,15 @@
+import ldap
+from django.core.exceptions import ValidationError
+
+from dienst2.settings import LDAP_HOST, LDAP_USER_DN
+
+
+def validate_ldap_username(username):
+    conn = ldap.initialize(LDAP_HOST)
+    dn = LDAP_USER_DN % ldap.dn.escape_dn_chars(username)
```

Comment:
hmmm... volgens mij wordt deze manier van string formatten bijna niet meer gebruikt, geen idee of het deprecated is ofzo. Ik zou verwachten dat `LDAP_USER_DN="uid={},ou=People,dc=ank,dc=chnet"` en dan `LDAP_USER_DN.format(username)`

Reply:
Aangepast!